spec/features/associate_twitter_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'spec_helper'
+
+describe "Twitter association" do
+
+  context "can link to Twitter" do
+
+    before do
+      visit "/users/new"
```

Comment:
This can be taken care of with the `login_as()` helper in `spec/support/authentication.rb